docs/python/how-to-set-workflow-options-in-python.md

```diff
@@ -0,0 +1,41 @@
+---
+id: how-to-set-workflow-options-in-python
+title: How to set Workflow options in Python
+sidebar_label: Set Workflow options
+description: Set Workflow options
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+Create the following script at `start_workflow.py`:
```

Comment:
Why am I creating a whole script with a main and everything just to demonstrate setting workflow options. In the rest of the guide, you just include relevant snippets. Also, this won't work anyways because there are no imports.